cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,58 @@
 /// <reference types='cypress' />
 
+const { generateUser, generateIdOfInput } = require('../support/generate');
+
 describe('Student Registration page', () => {
   before(() => {
-
+    cy.visit('https://demoqa.com/automation-practice-form');
   });
 
-  it('', () => {
-
+  it('modal window should contain all the inputed data', () => {
+    const user = generateUser();
+    cy.get(generateIdOfInput('firstName'))
```

Comment:
Is it really more convenient than just using `#`? :D
You don't have to change that if you don't want to, but it looks strange.